utilities/pipelines/deploymentRemoval/Clear-ManagementGroupDeployment.ps1

```diff
@@ -7,11 +7,14 @@ Bulk delete all deployments on the given management group scope
 Bulk delete all deployments on the given management group scope
 
 .PARAMETER ManagementGroupId
-Mandatory. The Resource ID of the Management Group to remove the deployments for.
+Mandatory. The Resource ID of the Management Group to remove the deployments from.
 
 .PARAMETER DeploymentStatusToExclude
 Optional. The status to exlude from removals. Can be multiple. By default, we exclude any deployment that is in state 'running' or 'failed'.
 
+.PARAMETER maxDeploymentRetentionInDays
+Optional. The time to keep deployments for beyong the deployment statuses to exclude. In other words, if a deployment is older than the threshold, they will always be deleted.
```

Comment:
```suggestion
Optional. The time to keep deployments with a status to exclude. In other words, if a deployment is in a status to exclude, but older than the threshold, it will be deleted.
```